### ISSUE TITLE
The code example for trim ltrim and rtrim from the manual do not work

### ISSUE THREAD

**Issue by riknor** [jqlang/jq#3127]
This example from the manual do not work 
https://jqplay.org/jq?q=trim%2C%20ltrim%2C%20rtrim&j=%22%20abc%20%22

![image](https://github.com/jqlang/jq/assets/859655/74ea73e2-3c7c-41bf-a09f-e22bb0c5a350)

**Comment by wader** [comment]
Hello, dup of https://github.com/jqlang/jq/issues/3078 and probably another reason why the online documentation should point to the latest release instead of development version 🤔

If you need a trim function in latest stable release you can define one something like this:
```sh
$ jq -n 'def trim: capture("^\\s*(?<s>.*?)\\s*$"; "m").s; " abc " | trim'
"abc"
```

**Comment by wader** [comment]
Now fixed by #3130 but /dev manual will still have possibly broken jqplay examples
